target/cheri-common/cheri_tagmem.c

```diff
@@ -96,20 +114,13 @@ static inline size_t num_tagblocks(RAMBlock* ram)
     return result;
 }
 
-#define TAGMEM_USE_BITMAP 1
-#if TAGMEM_USE_BITMAP
-// Use one bit per tag:
+_Static_assert(CAP_TAG_GET_MANY_SHFT < 3 || sizeof(long) >= 8,
+               "Otherwise settags cannot be atomic");
```

Comment:
This doesn't seem right... 64-bit atomics exist on some 32-bit architectures, and having an unbounded CAP_TAG_GET_MANY_SHFT on 64-bit architectures doesn't make sense.

Reply:
I think this assert is just unnecessary tbh. Something like it might appear when we merge the changes for concurrent tags, but it serves no purpose here, so I will just delete it.